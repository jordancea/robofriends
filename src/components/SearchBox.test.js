import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import SearchBox from "./SearchBox.js";

it("should render the SearchBox component", () => {
  const mochSearchChange = jest.fn();
  const wrapper = shallow(<SearchBox searchChange={mochSearchChange} />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
