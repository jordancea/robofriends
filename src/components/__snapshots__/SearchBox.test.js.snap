// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the SearchBox component 1`] = `
<div
  className="pa2"
>
  <input
    aria-label="Search Robots"
    className="pa3 ba b--green bg-lightest-blue"
    onChange={[MockFunction]}
    placeholder="search robots"
    type="search"
  />
</div>
`;
